easybuild/easyconfigs/m/mkl-service/mkl-service-2.3.0-foss-2020b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'mkl-service'
+version = '2.3.0'
+
+homepage = 'https://github.com/IntelPython/mkl-service'
+description = "Python hooks for Intel(R) Math Kernel Library runtime control settings."
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

Comment:
@dithwick I'm half-wondering whether this should in fact be using the `gomkl` toolchain?

Reply:
Yeah it probably could do come to think of it. I think I just put it under foss to keep it consistent with the intel chain but I'll try moving it.

Reply:
Actually thinking about it, wouldn't that then force Keras on to gomkl as well or have I misunderstood the hierarchy? I want to avoid doing that as otherwise Tensorflow etc. will also need to be built on gomkl.